src/Adaptor.js

```diff
@@ -45,15 +44,20 @@ export function execute(...operations) {
  */
 function login(state) {
   const { url, user, password } = state.configuration;
+  const body = JSON.stringify({
+    user: {
+      user_name: user,
+      password,
+    },
+  });
 
   const params = {
     method: 'POST',
-    url: `${url}/api/login`,
-    json: {
-      user_name: user,
-      password,
+    url: `${url}/api/v2/tokens`,
+    headers: {
+      'Content-Type': 'application/json',
     },
-    jar: true,
+    body,
```

Comment:
Three things to discuss: (1) I'm not 100% sure that this abstraction is useful. Since `body` is not re-used, why not leave it in the `params` object? (2) The `json` key is being replaced with a key called `body`. Is this desired? (3) Why not more `jar`?